.github/workflows/ci.yml

```diff
@@ -185,22 +162,29 @@ jobs:
           CI_BRANCH: ${{ github.ref	}}
           CI_BRANCH_BASE: refs/heads/main
 
-  deploy-strapi-main:
-    name: Deploy Strapi on Heroku production
-    if: contains( github.ref, 'main')
+  deployment-chromatic-pr:
```

Comment:
Can you explain the changes here, why chromatic is impacted for example ?

Reply:
Ok I dived a bit deeper in it. I'll accept it even though I have these questions:

 - Why does deployment-chromatic-main depends on bump-version ? 
 - Why does deploy-strapi depends on bump version ?
 
 I understand why deploy-web-app depends on bump-version of the web-app, but not sure about the others

Reply:
`bump-version` performs changes, commit them and pushes them. 

I made a small change on the `deploy-X-main` jobs. The checkout action now doesn't checkout the commit which triggered the pipeline (default behaviour, but the latest commit from main). This way it checkouts the commit created by `bump-version`. 

I did this to make sure that the deployed App version would correspond to the commit bumping the version. 

Maybe it's overkill for strapi and chromatic. I think it would make it more straightforward for debugging. If someone complains about something being bugged in version X, you just have to find the corresponding commit on Strapi/Chromatic.

Reply:
If you don't think that this is necessary, I can easily change that. I will also speed up the CI.